src/Compilers/VisualBasic/Portable/Parser/BlockContexts/SingleLineElseContext.vb

```diff
@@ -92,34 +93,90 @@ EndBlock:
                     End If
                     GoTo EndBlock
 
+                Case Else
+                    Throw ExceptionUtilities.UnexpectedValue(token.Kind)
```

Comment:
Am I right in assuming this fail safe would've caught this case and prevented a successful compilation?